roles/ceilometer-common/templates/etc/ceilometer/ceilometer.conf

```diff
@@ -53,13 +53,15 @@ host = 0.0.0.0
 port = {{ endpoints.ceilometer.port.backend_api }}
 
 [keystone_authtoken]
-identity_uri = {{ endpoints.identity_uri }}
-auth_uri = {{ endpoints.auth_uri }}
-admin_tenant_name = service
-admin_user = ceilometer
-admin_password = {{ secrets.service_password }}
+auth_url = {{ endpoints.keystone.url.admin }}
```

Comment:
Should this be using the keystone.url.public value for auth_url? 
It looks like the "identity_uri" config option should be pointed to the admin value.  (Referencing the mitaka config guide: http://docs.openstack.org/mitaka/config-reference/telemetry/telemetry_service_config_opts.html)